hack/test-integration.sh

```diff
@@ -55,10 +55,14 @@ testexec="${testdir}/${name}.test"
 mkdir -p "${testdir}"
 
 # build the test executable (cgo must be disabled to have the symbol table available)
-pushd "${testdir}" &>/dev/null
-echo "Building test executable..."
-CGO_ENABLED=0 go test -c -tags="${tags}" "${OS_GO_PACKAGE}/${package}"
-popd &>/dev/null
+if [[ -n "${OPENSHIFT_SKIP_BUILD:-}" ]]; then
```

Comment:
yeah, I'd like env vars across test suits to be consistent.

On Tue, May 31, 2016 at 10:42 PM, Maru Newby <notifications@github.com>
wrote:

> In hack/test-integration.sh
> <https://github.com/openshift/origin/pull/8287#discussion_r65294861>:
>
> > @@ -55,10 +55,14 @@ testexec="${testdir}/${name}.test"
> >  mkdir -p "${testdir}"
> >
> >  # build the test executable (cgo must be disabled to have the symbol table available)
> > -pushd "${testdir}" &>/dev/null
> > -echo "Building test executable..."
> > -CGO_ENABLED=0 go test -c -tags="${tags}" "${OS_GO_PACKAGE}/${package}"
> > -popd &>/dev/null
> > +if [[ -n "${OPENSHIFT_SKIP_BUILD:-}" ]]; then
>
> Can do.
>
> The dind deployment script uses OPENSHIFT_SKIP_BUILD to indicate that the
> deployment should take place without building the image or openshift, and
> the extended networking test runner uses the same flag to both skip
> building the test and control deployment. I think the name should remain
> for the dind script, but would you like to see the test runner use
> TEST_ONLY for consistency?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/8287/files/5eb01fdef0edc21fd5fe921c1e3015ddecd6bbac..e324a228a2ab6c2cc7fdff8881a86dbbdf3b0a78#r65294861>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p_RYkY8omQmN4wCGZ8IcU_7oqJtqks5qHPGygaJpZM4H7j2i>
> .
>